lib/common/options.c

```diff
@@ -954,14 +954,30 @@ pcmk__cluster_option(GHashTable *options, const char *name)
  * \param[in]     filter      If not \c pcmk__opt_context_none, include only
  *                            those options whose \c context field is equal to
  *                            \p filter
+ * \param[in]     table       Table from which to get options
+ *
+ * \return Standard Pacemaker return code
  */
-void
-pcmk__cluster_option_metadata(pcmk__output_t *out, const char *name,
-                              const char *desc_short, const char *desc_long,
-                              enum pcmk__opt_context filter)
+int
+pcmk__format_option_metadata(pcmk__output_t *out, const char *name,
+                             const char *desc_short, const char *desc_long,
+                             enum pcmk__opt_context filter,
+                             enum pcmk__opt_table table)
```

Comment:
slightly annoyed that I named the enum `pcmk__opt_table` while I'm usually calling the variables `option_list` or similar. Might rename. Similar in an informal sense, but we're used to `GHashTable`s and `GList`s.